src/main/zapHomeFiles/hud/drawer.js

```diff
@@ -12,19 +12,73 @@ Vue.component('history', {
     template: '#history-template',
     data() {
         return {
-            messages: []
+            filter: '',
+            regexEnabled: false,
+            messages: [],
+            hiddenMessageCount: 0,
+            isRegExError: false,
+            enableRegExText: I18n.t("history_enable_regex")
         }
     },
     computed: {
         timeDescendingMessages() {
             return this.messages.slice().reverse();
+        },
+        messageCount() {
+            return this.messages.length;
+        },
+        filteredMessages() {
+            const self=this,
+                  isRegex = this.regexEnabled;
+            let re;
+
+            if (isRegex){
+                try {
+                    re = new RegExp(this.filter);
+                    this.isRegExError = false;
+                }
+                catch (ex) {
+                    this.isRegExError = true;
+                    return []; //Return empty array if invalid RegEx
+                }
+            }
+
+            return this.messages.filter( message => {
+                if (self.filter.trim().length === 0) {
+                    return true;
+                }
+                if (isRegex){
+                    return re.test(message.url);
+                }
+                else{
+                    return message.url.indexOf(self.filter)>=0; 
+                }
+            });
+        },
+        historyItemsFiltered() {
+            let historyItemsFilteredString = I18n.t("history_items_filtered")
+                    .replace('{0}', this.hiddenMessageCount)
```

Comment:
With latest changes (merge upstream) these can be passed as parameters to `t`, like in:
https://kazupon.github.io/vue-i18n/guide/formatting.html#list-formatting

Reply:
Validated that works! I had tried that method previously but the interpolation wasn't working before. Merging upstream fixed, thanks for heads up, felt wrong. 👍